docs/reference/configuration/mesh.md

```diff
@@ -53,11 +53,12 @@ Option name | Features supported | Estimated binary size of Nanostack
 
 If you want to optimize the flash usage, you need to configure Nanostack. The configuration to choose depends mostly on the preferred use case.
 
-See the [6LoWPAN technology overview](mesh-tech.html) for the definition of star and mesh networks. These same principles apply also to the Thread protocol.
+See the [mesh technology overview](mesh-tech.html) for the definition of star and mesh networks. These same principles apply also to the Wi-SUN and Thread protocols.
```

Comment:
mesh-tech now contains mainly 6LoWPAN_ND related information

Reply:
star and mesh network definitions are at mesh-tech page, so no need to update this, but I'll update rest of the links